ui/raidboss/data/06-ew/raid/p10n.ts

```diff
@@ -1,14 +1,201 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  silkspitOnYou: boolean;
+}
+
+const bossNameUnicode = 'Pand\u00e6monium';
 
 const triggerSet: TriggerSet<Data> = {
   id: 'AnabaseiosTheTenthCircle',
   zoneId: ZoneId.AnabaseiosTheTenthCircle,
   timelineFile: 'p10n.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      silkspitOnYou: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'P10N Silkspit You',
+      type: 'HeadMarker',
+      netRegex: { id: '01CE' },
+      condition: (data, matches) => data.me === matches.target,
+      alertText: (data, _matches, output) => {
+        data.silkspitOnYou = true;
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from party and posts',
+        },
+      },
+    },
+    {
+      id: 'P10N Silkspit Others',
+      type: 'HeadMarker',
+      netRegex: { id: '01CE', capture: false },
+      delaySeconds: 0.2,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.silkspitOnYou === false)
+          return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from marked players',
+        },
+      },
+    },
+    {
+      id: 'P10N Pandaemonic Pillars',
+      type: 'StartsUsing',
+      netRegex: { id: '825D', source: bossNameUnicode, capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Soak towers',
```

Comment:
I think we've called this `Get Towers` in so many places that it should probably be a `Response` or `Output` at this point (not in this PR, but just talking out loud).

Reply:
No time like the present?  I pulled a full set of locale strings from hades-ex, and I'll push this in the next commit.